unlock-app/src/__tests__/services/storageService.test.js

```diff
@@ -57,7 +57,7 @@ describe('StorageService', () => {
 
   describe('lockLookUp', () => {
     describe('when the requested lock exists', () => {
-      it('returns the details', done => {
+      it.todo('returns the details', done => {
```

Comment:
It looks like the todo here is causing the test to fail.  We should be able to keep this test if the portions around the name assertion are removed and we ensure that the address is present on the mocked data

Reply:
oopsie! removing it now.